src/posthog-core.ts

```diff
@@ -813,6 +813,8 @@ export class PostHog {
             properties['$window_id'] = windowId
         }
 
+        properties['$using_proxy'] = this.requestRouter.isUsingProxy()
```

Comment:
for cloud this name is correct (I think)... but for self hosted they might not be using a proxy but won't have app.posthog.com

so this is really `has_set_api_host` that's at least true for self-hosted if not useful for self-hosted

Reply:
That's a good point.

Reply:
Actually I'm wondering if we want to actually `include` the host... that way we can more easily later debug issues with it...
```suggestion
        if (this.requestRouter.isUsingProxy()) {
	            properties['$lib_custom_api_host'] = this.config.api_host
	}
	```

Reply:
@raquelmsmith has the same idea about including the actual host so we have that data directly. I think this is a good route if infra can't set it as you've mentioned below. Let me check with them and then circle back here.